warehouse/templates/manage/projects.html

```diff
@@ -27,7 +27,11 @@ <h1 class="page-title">Your projects ({{ projects|length }})</h1>
     <div class="package-snippet">
       <div class="split-layout split-layout--table split-layout--wrap-on-tablet">
         <div>
-          <h3 class="package-snippet__title">{{ project.name }}</h3>
+          <h3 class="package-snippet__title">{{ project.name }}
+            {% if project.name in projects_sole_owned %}
+            <span class="badge">SOLE OWNER</span>
```

Comment:
I don't think we need this to be "SOLE OWNER" as we have a CSS style that will make it uppercase.  It should instead be "Sole owner".

It would also be great, if we could put a 4px left margin on this.  If you could follow the same pattern as shown in this pr: https://github.com/pypa/warehouse/pull/4383/files

... that would be great :)

Let me know if this isn't clear!

Reply:
Done, it now looks like this:
<img width="794" alt="screen shot 2018-07-29 at 13 35 08" src="https://user-images.githubusercontent.com/16256035/43365353-59ed99e0-9334-11e8-86f5-98c9f4ca0e54.png">

Reply:
great :) could you also take a screenshot on a smaller resolution (mobile) ?

Reply:
Sure, on mobile resolution it's like this:
<img width="460" alt="screen shot 2018-07-29 at 14 03 05" src="https://user-images.githubusercontent.com/16256035/43365626-49c76eac-9338-11e8-9471-46fa97a58d05.png">